Frameworks/QuartzCore/CALayer.mm

```diff
@@ -1570,9 +1568,9 @@ - (CATransform3D)sublayerTransform {
 */
 - (void)setBackgroundColor:(CGColorRef)color {
     if (color != nil) {
-        priv->backgroundColor = *[static_cast<UIColor*>(color) _getColors];
+        priv->backgroundColor = *(reinterpret_cast<const __CGColorQuad*>(CGColorGetComponents(color)));
```

Comment:
that is not the scope of this PR.The main purpose to detangle CGColor<->UIColor dependencies 
We should do that at some point, Raj could weight in on the priority of this.

---
In reply to: [113504346](https://github.com/Microsoft/WinObjC/pull/2538#discussion_r113504346) [](ancestors = 113504346)